app/main/views/suppliers.py

```diff
@@ -541,8 +541,14 @@ def find_supplier_services(supplier_id):
 
     frameworks = data_api_client.find_frameworks()['frameworks']
     supplier = data_api_client.get_supplier(supplier_id)["suppliers"]
-    services = data_api_client.find_services(supplier_id)['services']
-
+    # TODO replace this temporary fix for DOS2 when a better solution has been created.
+    services = data_api_client.find_services(
+        supplier_id=supplier_id,
+        framework=','.join(
+            [f['slug'] for f in frameworks if f['status'] == 'live'
+                or f['slug'] == 'digital-outcomes-and-specialists-2']
+        )
+    )['services']
```

Comment:
A heads up that this view will display the service status as well:

![screenshot-localhost-2018 10 05-09-51-03](https://user-images.githubusercontent.com/3492540/46525694-38dc4c80-c884-11e8-809a-e465f01ca02f.png)

If the services themselves are still 'public' that's what CCS Category will see (so they can edit them as usual). Would this be confusing? Our different types of admin users should know that DOS services aren't actually visible to buyers regardless of framework status, right?

Reply:
Hmm...I see what you mean. I **hope** that they realise that in this sense "public" means "active" or "viable" or something. 

It feels like another thing pointing towards the idea of having a new frameworks status in between live and expired.

I think for now it's fine, as this is just a short term thing to allow them to do what they need to do, and they are experts in the domain so should understand what's going on.

Reply:
Well I guess we'll find out 😸 

Agreed on the need for a new status. `sunset`?

Reply:
Ooo....that's a good one. 

I was leaning towards `cooldown`, but `sunset` is much more picturesque.